src/util/runPluginFunction.js

```diff
@@ -0,0 +1,51 @@
+/* eslint global-require:"off" */
+/**
+ * calls a specific plugin function in any plugins registered in .arc
+ */
+module.exports = function runPluginFunction(params, functionName) {
```

Comment:
Pulled out this code into a new file that can be shared between the deploy and create commands.